src/proxy/libp2p_transport_proxy.erl

```diff
@@ -28,12 +27,9 @@ start_listener(_Pid, _Addr) ->
 match_addr(Addr, _TID) when is_list(Addr) ->
     false.
 
--spec sort_addrs([string()]) -> [string()].
+-spec sort_addrs([string()]) -> [{integer(), string()}].
 sort_addrs(Addrs) ->
-    Addrs.
-
--spec priority() -> integer().
-priority() -> 99.
+    [{5, A} || A <- Addrs].
```

Comment:
Why not keep this as 99?

Reply:
idk just wanted a smaller range  99 seemed just useless

Reply:
I guess this will never pass `match_addr` so it's academic

Reply:
yes